config.yaml

```diff
@@ -4,6 +4,21 @@ options:
     default: true
     description: Restrict access to sensitive files/dirs (/proc/sched_debug, /sys/kernel/slab)
 
+  lxd-listen-bgp:
+    type: boolean
+    default: false
+    description: Enable/disable the BGP endpoint
+
+  lxd-listen-https:
+    type: boolean
+    default: false
+    description: Enable/disable the remote API (HTTPS)
+
+  lxd-listen-metrics:
```

Comment:
If not too much work to skip, it may be worth waiting until metrics are merged before adding this one?

Reply:
np, done.